public/js/utils.js

```diff
@@ -6,13 +6,14 @@ async function enableEthereum () {
       await window.ethereum.enable()
       console.log('Ethereum enabled...')
     } catch (error) {
-      console.error(error)
+      alert(error)
+      metamask = undefined
     }
   } else if (window.web3) {
-    console.log('Legacy web3 detected!')
+    alert('Legacy web3 detected!')
```

Comment:
I think this isn't an error and should have no popup.

Reply:
fixed